core_lib/src/tool/stroketool.h

```diff
@@ -66,6 +68,13 @@ public slots:
     void onViewUpdated();
 
 protected:
+    QPointF getCurrentPressPixel() const;
+    QPointF getCurrentPressPoint() const;
+    QPointF getCurrentPixel() const;
+    QPointF getCurrentPoint() const;
+    QPointF getLastPixel() const;
+    QPointF getLastPoint() const;
```

Comment:
I know that currently it's not like this but if we're moving things around anyway, then we might as well improve it a bit.
```suggestion
    const QPointF& getCurrentPressPixel() const;
    const QPointF& getCurrentPressPoint() const;
    const QPointF& getCurrentPixel() const;
    const QPointF& getCurrentPoint() const;
    const QPointF& getLastPixel() const;
    const QPointF& getLastPoint() const;
```

Reply:
Ah you're right, because it's being mapped first. 💡